src/Reader/ReaderT.js

```diff
@@ -1,9 +1,12 @@
 /** @license ISC License (c) copyright 2017 original and current authors */
 /** @author Ian Hofmann-Hicks (evil) */
 
+const VERSION = 1
```

Comment:
What is the main goal of `VERSION` here? isn't it better to put it in another file that others can use the same version?

Reply:
That is to report the version of the specific type for the Sanctuary Type siggy. So each type may have its own VERSION, for instance if you look at `Pair` that is a version 2 now, because the API changed. I was thinking that it would make sense that that stays with the type in the file.

What are you suggesting? Like a lookup table?

Reply:
I thought was the same version for each one. So yeah, put it in the same file its ok.